### PR TITLE
DOC: Improve ndarray.ctypes example

### PR DESCRIPTION
**Pull request by sethtroisi**

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]
Looks reasonable. @mattip, does refguide find these?

**Comment by sethtroisi** [comment]
^ @eric-wieser  I suspect it does not (because I would think it would fall to validate the random memory address and they previous code didn't define x)

File: numpy/core/_add_newdocs.py [hunk 1]
```diff
@@ -2025,25 +2025,22 @@
...
-    30439712
-    >>> x.ctypes.data_as(ctypes.POINTER(ctypes.c_long))
-    <ctypes.LP_c_long object at 0x01F01300>
-    >>> x.ctypes.data_as(ctypes.POINTER(ctypes.c_long)).contents
-    c_long(0)
-    >>> x.ctypes.data_as(ctypes.POINTER(ctypes.c_longlong)).contents
-    c_longlong(4294967296L)
+    31962608
+    >>> x.ctypes.data_as(ctypes.POINTER(ctypes.c_uint32))
+    <__main__.LP_c_uint object at 0x7ff2fc1fc200>
```
Comment (charris) [inline numpy/core/_add_newdocs.py]: These outputs will vary, IIRC there is a way to mark that. @mattip Thoughts?

Comment (charris) [reply numpy/core/_add_newdocs.py]: Use a trailing `# may vary` for those lines.

**Comment by mattip** [comment]
Strange, I wonder if it has to do with refguide-check running on macOS on azure, maybe that python does not have ctypes?

**Comment by mattip** [comment]
Thanks @sethtroisi

**Pull request merged**